package.json

```
@@ -732,34 +732,42 @@
 		"version:changesets": "changeset version --global-changelog",
 		"push": "tsx ./dual-publish.ts",
 		"vscode:prepublish": "pnpm build",
-		"build": "pnpm types && node esbuild.mjs",
+		"build": "pnpm types && node ./scripts/esbuild.mjs",
 		"compile": "cross-env NODE_ENV=0 pnpm build",
 		"package": "cross-env NODE_ENV=1 pnpm build",
 		"lint": "eslint --fix",
-		"test": "npm-run-all -p test:* --continue-on-error",
+		"test": "npm-run-all -s test:* --continue-on-error",
 		"postinstall": "husky",
 		"types": "npm-run-all -p build:*:tsc",
 		"types:watch": "npm-run-all -p watch:*:tsc",
 		"build:web": "npm-run-all -s build:web:*",
 		"build:desktop": "npm-run-all -s build:desktop:*",
 		"build:web:tsc": "tsc --noEmit --project tsconfig.web.json",
-		"build:web:esbuild": "node esbuild.mjs --mode web",
+		"build:web:esbuild": "node ./scripts/esbuild.mjs --mode web",
 		"build:desktop:tsc": "tsc --noEmit --project tsconfig.app.json",
-		"build:desktop:esbuild": "node esbuild.mjs --mode desktop",
+		"build:desktop:esbuild": "node ./scripts/esbuild.mjs --mode desktop",
 		"watch:web": "npm-run-all -p watch:web:*",
 		"watch:desktop": "npm-run-all -p watch:desktop:*",
-		"watch:web:esbuild": "node esbuild.mjs --watch --mode web",
+		"watch:web:esbuild": "node ./scripts/esbuild.mjs --watch --mode web",
 		"watch:web:tsc": "tsc --noEmit --watch --project tsconfig.web.json",
-		"watch:desktop:esbuild": "node esbuild.mjs --watch --mode desktop",
+		"watch:desktop:esbuild": "node ./scripts/esbuild.mjs --watch --mode desktop",
 		"watch:desktop:tsc": "tsc --noEmit --watch --project tsconfig.app.json",
-		"test:web": "npm run build:web && npm-run-all -s test:web:*",
-		"test:desktop": "npm run build:desktop && npm-run-all -s test:desktop:*",
+		"test:web": "pnpm run build:web && npm-run-all -s test:web:*",
 		"test:web:tsc": "tsc --noEmit --project test-tsconfigs/tsconfig.web.json",
-		"test:web:esbuild": "node esbuild.mjs --mode web --test",
-		"test:web:vscode": "vscode-test --coverage --label webUnitTest",
+		"test:web:esbuild": "node ./scripts/esbuild.mjs --mode web --test",
+		"test:web:vscode": "vscode-test --coverage --label webUnitTest --coverage-output ./web-coverage",
```

Change name format from `<platform>-coverage` to `coverage-<platform>` so they are all next to each other in the file browser.
```suggestion
		"test:web:vscode": "vscode-test --coverage --label webUnitTest --coverage-output ./coverage-web",
```